tests/test_05_l2vpn.py

```diff
@@ -267,7 +269,8 @@ def test_050_delete_l2vpn_successfully(self):
         ## -> tenet
         response = requests.get("http://tenet:8181/api/kytos/mef_eline/v2/evc/")
         assert len(response.json()) == 0, response.text
-    
+
+    @pytest.mark.xfail(reason="AssertionError: assert ', 0% packet loss, ... 100% packet loss,...'")
```

Comment:
@gretelliz can you please add a comment on the code referring to the issue number that is related to this problem?